lms/djangoapps/commerce/api/v1/models.py

```diff
@@ -88,8 +88,7 @@ def update(self, attrs):
             merged_mode.min_price = posted_mode.min_price
             merged_mode.currency = posted_mode.currency
             merged_mode.sku = posted_mode.sku
-            if posted_mode.expiration_datetime is not None:
```

Comment:
And actually, it looks like we have code that explicitly allows the second case (verification deadline, no upgrade deadline) here: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/commerce/api/v1/serializers.py#L71-L86.